maia-core/src/interval.rs

```diff
@@ -0,0 +1,171 @@
+use bit_vec::BitVec;
```

Comment:
🤔 I think this should have been moved rather than created as a new file? Perhaps you forgot to delete the original in `maia`?

Reply:
spot on, that turned out to be the reason. removed now.